api/RadioNodeList.json

```diff
@@ -11,7 +11,7 @@
             "version_added": null
           },
           "edge": {
-            "version_added": null
+            "version_added": true
```

Comment:
Confirmed by https://software.hixie.ch/utilities/js/live-dom-viewer/saved/6828 in Edge 18 using a very different approach. Wasn't in Edge 13 but I won't find the exact version in this PR.